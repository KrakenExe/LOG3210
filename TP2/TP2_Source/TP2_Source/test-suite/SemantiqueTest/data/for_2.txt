Declaration
num a;
EndDeclaration

for(c = 0; a < 1; i = i + 1) {
    a = 1 + 1;
}