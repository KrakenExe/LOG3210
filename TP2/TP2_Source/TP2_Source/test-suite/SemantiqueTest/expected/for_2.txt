Invalid use of undefined Identifier c